source/qb64pe.bas

```diff
@@ -12577,7 +12577,10 @@ END IF
 'Clear nm output from previous runs
 FOR x = 1 TO ResolveStaticFunctions
     IF LEN(ResolveStaticFunction_File(x)) THEN
-        s$ = MakeNMOutputFilename$(ResolveStaticFunction_File(x))
+        s$ = MakeNMOutputFilename$(ResolveStaticFunction_File(x), 0)
```

Comment:
Not sure if we should do a `ClearBuffers` here too, just in case the file was already loaded into memory. It can definitly not make things worst, but maybe better :), as it ignore calls with for unknown buffers (see utilities\s-buffer\sb_qb64pe_extension.bm).
i.e. instead of `IF _FILEEXISTS(s$) THEN KILL s$` write `IF _FILEEXISTS(s$) THEN KILL s$: ClearBuffers s$`